ts_export/src/process.rs

```diff
@@ -93,12 +94,14 @@ impl ProcessModule {
                     arguments: PathArguments::None,
                 });
                 match item_mod.content {
-                    Some((_, items)) => Some(ProcessModule::new(path, items, "crate")),
-                    _ => process_spawner.create_process(path),
+                    Some((_, items)) => Some(Ok(ProcessModule::new(path, items, "crate"))),
+                    _ => process_spawner.create_process(path).map_err(|e| e.into()).invert()
```

Comment:
This is where new `result` dependency is used. 
If you don't like having an extra dep, we can rewrite it manually, it will be just nosier